fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/indexes/TextIndexTest.java

```diff
@@ -1329,14 +1330,14 @@ public void invalidScans() throws Exception {
         LOGGER.info(KeyValueLogMessage.of("planned query",
                         TestLogMessageKeys.QUERY, query,
                         LogMessageKeys.PLAN, plan,
-                        TestLogMessageKeys.PLAN_HASH, plan.planHash()));
+                        TestLogMessageKeys.PLAN_HASH, plan.planHash(PlanHashable.PlanHashKind.LEGACY)));
```

Comment:
Let's just leave these calls in test cases that don't directly have anything to do with the work done in this PR to call `planHash()`

Reply:
Done